src/style.css

```diff
@@ -1 +1,69 @@
 /* Styles go here */
+* {
+  padding: 0;
+  margin: 0;
```

Comment:
add these styles only for needed selectors
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/5523ae9e-b122-40d7-b1f3-f31449b14f2e)